src/components/vm/vmMigrateDialog.jsx

```diff
@@ -71,7 +71,7 @@ const DurationRow = ({ temporary, setTemporary }) => {
                                         <p>{_("A copy of the VM will run on the destination and will disappear when it is shut off. Meanwhile, the origin host keeps its copy of the VM configuration.")}</p>
                                     </FlexItem>
                                 </Flex>}>
-                           <button onClick={e => e.preventDefault()} className="pf-c-form__group-label-help">
+                           <button onClick={e => e.preventDefault()} className="pf-v5-c-form__group-label-help">
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1088-20230630-122011-699fe1a0-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)